src/AspNetCoreModuleV2/CommonLib/FileOutputManager.cpp

```diff
@@ -62,7 +60,12 @@ FileOutputManager::Start()
 
 
     // TODO fix string as it is incorrect
```

Comment:
TODO?

Reply:
Removed in other PR: https://github.com/aspnet/IISIntegration/pull/1332/files#diff-f02943abd05fa834ec7f4786668012a6L66